cmd/dep/main.go

```diff
@@ -54,9 +59,9 @@ type Config struct {
 }
 
 // Run executes a configuration and returns an exit code.
-func (c *Config) Run() (exitCode int) {
+func (c *Config) Run() int {
 	// Build the list of available commands.
-	commands := []command{
+	commands := [...]command{
```

Comment:
Can you point me to some link or maybe explain what this does? Never seen `[...]` before. Just curious. 😅

Reply:
@darkowlzz scroll a little bit down to the array section :) https://golang.org/ref/spec#Composite_literals